pub/index.php

```diff
@@ -10,7 +10,7 @@
 use Magento\Framework\App\Filesystem\DirectoryList;
 
 try {
-    require realpath(__DIR__) . '/../app/bootstrap.php';
```

Comment:
It make sense to change the code in the following way:
`realpath(__DIR__ . '/../app/bootstrap.php')`
in this case we will get `/root/folder/app/bootstrap.php` insead of `/root/folder/pub/../app/bootstrap.php`